src/main/java/apoc/mongodb/MongoDBColl.java

```diff
@@ -39,9 +39,30 @@ public void close() throws IOException {
         mongoClient.close();
     }
 
+    /**
+     * It translates a MongoDB document into a Map where the "_id" field is not an ObjectId
+     * but a simple String representation of it
+     *
+     * @param document
+     * @return
+     */
+    private Map<String, Object> documentToPackableMap(Map<String, Object> document) {
+        /**
+         * A document in MongoDB has a special field "_id" of type ObjectId
+         * This object is not "packable" by Neo4jPacker so it must be converted to a value that Neo4j can deal with
+         *
+         * If the document is not null we simply override the ObjectId instance with its string representation
+         */
+        if (document != null) {
+            document.put("_id", document.get("_id").toString());
```

Comment:
I don't think "_id" can be null in MongoDB but I can do a better NP check.
To be honest I don't remember if there is something in the mongo driver but I can check before updating the PR or if you want I can do this in a separate PR